### PR TITLE
[doc] `perldelta`: Minor template fixes

### PR DESCRIPTION
**Pull request by rwp0**
- http -> https links
- Linking modules and sections
- Apply formatting

### CODE REVIEW

**Comment by rwp0** [comment]
Finally managed to make a cleaner PR.

Please ignore the previous ones (which I closed).

File: pod/perldelta.pod [hunk 1]
```diff
@@ -58,7 +58,7 @@ future release, and will at that time need to be installed from CPAN.
 Distributions on CPAN which require these modules will need to list them as
 prerequisites.
 
-The core versions of these modules will now issue C<"deprecated">-category
+The core versions of these modules will now issue C<deprecated>-category
```
Comment (rwp0) [inline pod/perldelta.pod:61]: "" in addition to C<> seems redundant here

File: pod/perldelta.pod [hunk 2]
```diff
@@ -96,7 +96,7 @@ XXX
 =head1 Modules and Pragmata
 
 XXX All changes to installed files in F<cpan/>, F<dist/>, F<ext/> and F<lib/>
-go here.  If Module::CoreList is updated, generate an initial draft of the
+go here.  If L<Module::CoreList> is updated, generate an initial draft of the
```
Comment (rwp0) [inline pod/perldelta.pod:99]: Link to the module so it's documentation could be accesses

File: pod/perldelta.pod [hunk 3]
```diff
@@ -115,7 +115,7 @@ release manager will have to investigate the situation carefully.)
 
 =item *
 
-XXX Remove this section if Porting/corelist-perldelta.pl did not add any content here.
+XXX Remove this section if F<Porting/corelist-perldelta.pl> did not add any content here.
```
Comment (rwp0) [inline pod/perldelta.pod:118]: Mark it a file as with previous paragraph

File: pod/perldelta.pod [hunk 4]
```diff
@@ -142,7 +142,7 @@ before worked properly in Perl.
...
```
Comment (rwp0) [inline pod/perldelta.pod:145]: Ditto

File: pod/perldelta.pod [hunk 5]
```diff
@@ -192,8 +192,8 @@ include any changes in L<perldiag> that reconcile it to the C<C> code.
 
 =head2 New Diagnostics
 
-XXX Newly added diagnostic messages go under here, separated into New Errors
-and New Warnings
+XXX Newly added diagnostic messages go under here, separated into L</New Errors>
+and L</New Warnings>
```
Comment (rwp0) [inline pod/perldelta.pod:196]: Link to the sections, a practice done previously within the file with other sections too

File: pod/perldelta.pod [hunk 6]
```diff
@@ -362,7 +362,7 @@ XXX
 XXX Important bug fixes in the core language are summarized here.  Bug fixes in
 files in F<ext/> and F<lib/> are best summarized in L</Modules and Pragmata>.
 
-XXX Include references to GitHub issues and PRs as: [GH #12345] and the release
+XXX Include references to GitHub issues and PR's as: I<[GH #12345]> and the release
```
Comment (rwp0) [inline pod/perldelta.pod]: Emphasize the GH part, add ' to PR to better indicate the plural

File: pod/perldelta.pod [hunk 7]
```diff
@@ -398,7 +398,7 @@ XXX
 =item *
 
 XXX Add anything here that we forgot to add, or were mistaken about, in
-the perldelta of a previous release.
+the F<perldelta> of a previous release.
```
Comment (rwp0) [inline pod/perldelta.pod:401]: perldelta's are files, and each release has a file called `perldelta` by default (without version infix)

File: pod/perldelta.pod [hunk 8]
```diff
@@ -417,7 +417,7 @@ XXX Generate this with:
 
 If you find what you think is a bug, you might check the perl bug database
 at L<https://github.com/Perl/perl5/issues>.  There may also be information at
-L<http://www.perl.org/>, the Perl Home Page.
+L<https://www.perl.org/>, the Perl Home Page.
```
Comment (rwp0) [inline pod/perldelta.pod:420]: unfortunately all previous delta's went with http://, so fix this in the general template to affect all future perldeltas

**Review by rwp0 — COMMENTED** [review]
Made reviews on lines to clarify my points for modification

**Comment by Grinnz** [comment]
Can you also apply these changes to Porting/perldelta_template.pod?

File: pod/perldelta.pod (hunk 6 above)
Comment (Grinnz) [reply pod/perldelta.pod]: Apostrophes indicate possessive, not plural

**Comment by rwp0** [comment]
@Grinnz thanks, that's done

File: pod/perldelta.pod (hunk 6 above)
Comment (khwilliamson) [reply pod/perldelta.pod]: People sometimes use apostrophes in weird situations to indicate plural; I'm neutral about the practice.  In this case, I have a slight preference to not add it.

I however would like other feedback about italicizing the GH references.  I don't think it's worth the bother.

Comment (rwp0) [reply pod/perldelta.pod]: > I however would like other feedback about italicizing the GH references. I don't think it's worth the bother.

What about applying C<> instead?

Comment (rwp0) [reply pod/perldelta.pod]: > Apostrophes indicate possessive, not plural

Hmm, could use `-` here instead, it'd at least help separate the abbreviation

Comment (Grinnz) [reply pod/perldelta.pod]: Normally these tags get turned into links in the final release perldelta. Adding any other formatting here may encourage people to add that formatting when writing the perldelta and confuse things, so I would also side on omitting the formatting here.

Comment (hvds) [reply pod/perldelta.pod]: > People sometimes use apostrophes in weird situations to indicate plural; I'm neutral about the practice. In this case, I have a slight preference to not add it.

FWIW I've noticed fonts in which it is very hard to see that the 's' at the end of a pluralized abbreviation is lower-case, so it reads as PRS. An additional indication that a plural was intended could reduce confusion in such situations.

As far as formal correctness is concerned, I think classically the plural of PR (pull request) would be PR (pull requests).

> What about applying C<> instead?

That sounds like an excellent alternative.

File: Porting/perldelta_template.pod [hunk 9]
```diff
@@ -357,7 +357,7 @@ XXX
 XXX Important bug fixes in the core language are summarized here.  Bug fixes in
 files in F<ext/> and F<lib/> are best summarized in L</Modules and Pragmata>.
 
-XXX Include references to GitHub issues and PRs as: [GH #12345] and the release
+XXX Include references to GitHub issues and PR's as: I<[GH #12345]> and the release
```
Comment (jkeenan) [inline Porting/perldelta_template.pod]: I, too, believe italicizing the GH references is unnecessary (here and below).

**Review by jkeenan — CHANGES_REQUESTED** [review]

**Comment by rwp0** [comment]
Thanks for the feedback.

Reverted the changes concerning the apostrophe and formatting in regards to PR notation.

Please merge if all is good now.

**Comment by rwp0** [comment]
Not sure why reviews show a red X when I resolved the review myself. @jkeenan could you please re-review?

**Pull request merged**